src/scripts/nmdc_database_tools.py

```diff
@@ -229,7 +231,9 @@ def extract_study(ctx, study_id, output_file, quick_test, search_orphaned_data_o
     search_study_ids = [study_id]
     if search_legacy_identifiers:
         # add legacy study IDs to search
+        logger.info("Searching using current and legacy identifiers")
         search_study_ids.extend(study.get("gold_study_identifiers", []))
+        logger.info(f"Using study IDs: {search_study_ids}")
```

Comment:
why are we making a `search_study_ids` list with one item?

Reply:
If the `search-legacy-identifiers` option is selected, there could be >1 `study_id` (NMDC study ID and gold dtudy identifier(s))that we want to use as a search term for finding biosamples / omics processing.

Reply:
so does `--study-id` need to be configure to take multiple values and return a list? I think that would be good future proofing.

Reply:
I might lean the other way, actually. It's there right now only to support the ability to search `part_of` and `was_informed_by` using old (non-conformant) identifiers.  In normal cases we should not have any of these, so one could argue that once we get past the Re-ID-ing work (soon, I hope) we won't need either functionality anymore.  It's here as kind of a `look-for-relationships-that-should-not-exist` function for doing QC on our re-ID-ed studies

Reply:
Similarly for `--search-orphaned-data-objects`.  Normally, these things should not exist, so once we are done with this data cleaning exercise we can remove these options and their logic and leave `extract_study` cleaned up and simplified.